test/cppsuite/tests/base_test.cxx

```diff
@@ -36,5 +36,6 @@
  */
 class base_test : public test_harness::test {
     public:
-    base_test(const std::string &config, const std::string &name) : test(config, name) {}
+    base_test(const std::string &config, const std::string &wt_open_config, const std::string &name)
```

Comment:
We need to find a way to automate this process. We cannot perform this change if we have too many *.cxx tests.

Reply:
Either that or we create some kinda test_definitions struct?

Reply:
Yep sounds good, struct that contains config and wt_open_config for now.